supertokens_python/recipe/thirdparty/providers/custom.py

```diff
@@ -421,6 +422,11 @@ async def get_user_info(
                     self.config.user_info_endpoint, query_params, headers
                 )
 
+        if self.config.validate_access_token is not None and access_token is not None:
+            await self.config.validate_access_token(
+                access_token, self.config, user_context
+            )
```

Comment:
- In the node SDK, this check if before the above (`if access_token is not None and ...`) condition.
- In the node SDK, the condition for `if access_token is not None and ...` (above) is checking for `user_info_endpoint` not being None, but here we are checking for `config.token_endpoint` not being None, which is different. This may be a bug from before. Please see who wrote the code above (use git blame), and ask them about this.

Reply:
Updated the check to use `user_info_endpoint`.